docs/contribute/unit-tests.md

```diff
@@ -1,23 +1,83 @@
 
+## ⚙️ Command-Line Options
 
-|**Commands**|**Short Commands**|**Description**|**Example Usage**|
-|-|-|-|-|
-| **singlecompile**|`sc`| Compile a single `.c` file to `.wasm` and `.cwasm`. | `./wasmtest.sh sc file.c`|
-| **singlerun**|`sr`| Run a single previously compiled `.wasm`/`.cwasm`file. |`./wasmtest.sh sr file.c`|
-| **single** |`s`| Compile and run a single `.c` file in sequence (first compile to WASM, then run).| `./wasmtest.sh s file.c`|
-| **allcompile**| `ac`| Compile all `.c` test files in `$LIND_WASM_BASE/tests/unit-tests`. | `./wasmtest.sh ac`|
-| **allrun**|`ar`| Run all compiled tests in `$LIND_WASM_BASE/tests/unit-tests`.| `./wasmtest.sh ar`|
-| all|`a`| Compile and run all test files in `$LIND_WASM_BASE/tests/unit-tests`.| `./wasmtest.sh a`|
-| **filescompile**, **filecompile**|`fc`| Compile test files listed in a user-provided text file (each line should contain one `.c` filename).| `./wasmtest.sh fc filelist.txt`|
-| **filesrun**, **filerun**|`fr`| Run test files listed in a user-provided text file (assumes the `.wasm`/`.cwasm` files are already compiled). |`./wasmtest.sh fr filelist.txt`|
-| **files**, **file** |`f`| Compile and run test files listed in a user-provided text file (each line should contain one `.c` filename).| `./wasmtest.sh f filelist.txt`                             |
-| **`--timeout=<seconds>`** (optional)| *(none)*| Set a custom test timeout (in seconds). Default: **5 seconds**.| `./wasmtest.sh single file.c --timeout=10`|
+| **Option** | **Description** | **Example Usage** |
+|------------|------------------|--------------------|
+| `--skip <folder1> <folder2>` | Skip tests in the specified folders. | `./wasmtestreport.py --skip config_tests file_tests` |
+| `--run <folder1> <folder2>` | Run only the tests in the specified folders. | `./wasmtestreport.py --run config_tests file_tests` |
+| `--timeout <seconds>` | Timeout value in seconds for each test. Must be a positive integer. | `./wasmtestreport.py --timeout 10` |
+| `--output <filename>` | Output JSON file name (default: `results.json`). | `./wasmtestreport.py --output newresult` |
+| `--report <filename>` | Output HTML report file name (default: `report.html`). | `./wasmtestreport.py --report myreport.html` |
+| `--pre-test-only` | Only prepare the test files in Lind FS, no test execution. | `./wasmtestreport.py --pre-test-only` |
+| `--clean-testfiles` | Clean up the copied test files from Lind FS. | `./wasmtestreport.py --clean-testfiles` |
+| `--clean-results` | Clean all result files and outputs. | `./wasmtestreport.py --clean-results` |
 
+---
 
-### Notes
-- **`LIND_WASM_BASE`**: Defaults to `/home/lind/lind-wasm`. You can override it:  
-  ```bash
-  export LIND_WASM_BASE="/custom/path/to/lind-wasm"
-- **`<file>.c`**: A single test file to compile or run, found in `$LIND_WASM_BASE/tests/unit-tests`.  
-- **`<filelist>`**: A text file containing one `.c` filename per line, found in `$LIND_WASM_BASE/tests/unit-tests`.  
+## Testing Workflow
```

Comment:
To me "Testing Workflow" sounds like it contains instructions for a tester. But it seems like you're describing  `wasmtestreport.py` internals. This is probably helpful for a contributor, who wants to understand how the test suite works, but not necessarily for someone who just wants to run the tests. Could you maybe add some clarification?